demo/config/application.rb

```diff
@@ -16,6 +16,8 @@
 
 module Demo
   class Application < Rails::Application
+    config.load_defaults 6.0 if Rails.version.to_i >= 6
```

Comment:
What is this change?

Reply:
basically activates `zeitwerk` mode for the demo app
it only works for rails 6+ tho